stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -262,8 +262,7 @@ function sym_uplo(uplo::Char)
     end
 end
 
-
-@noinline throw_uplo() = throw(ArgumentError("uplo argument must be either :U (upper) or :L (lower)"))
+@noinline throw_uplo() = throw(ArgumentError("uplo argument must be either :U/'U' (upper) or :L/'L' (lower)"))
```

Comment:
@andreasnoack I guess this change here should be reverted according to your comment that the `Char` representation is internal? The additional checks in this PR should be fine, though.